bounded-collections/CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ The format is based on [Keep a Changelog].
 
 [Keep a Changelog]: http://keepachangelog.com/en/1.0.0/
 
+## [0.1.7] - 2023-05-05
+- Added `serde` feature, which can be enabled for no `std` deployments.
```

Comment:
Just to recapitulate: Feature `std` still enables `serde` (de)serialization and it is therefore not a breaking change.

Reply:
Regarding semver: I am flexible here :)

Reply:
No its all fine. just wanted to ensure that i understood it.